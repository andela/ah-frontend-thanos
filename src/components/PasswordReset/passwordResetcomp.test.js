import React from 'react';
import { shallow } from 'enzyme';
import NewPassword from './newPassword';
import PasswordReset from './index';

describe('<NewPassword />', () => {
  let newpasswordComponent;

  beforeEach(() => {
    newpasswordComponent = shallow(<NewPassword onChange={jest.fn()} onSubmit={jest.fn()} />);
  });

  it('renders the component', () => {
    expect(newpasswordComponent).toMatchSnapshot();
  });
});

describe('<PasswordReset />', () => {
  let newpasswordComponent;

  beforeEach(() => {
    newpasswordComponent = shallow(<PasswordReset />);
  });

  it('renders the component', () => {
    expect(newpasswordComponent).toMatchSnapshot();
  });
});
