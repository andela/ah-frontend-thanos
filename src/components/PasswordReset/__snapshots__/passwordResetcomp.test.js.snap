// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewPassword /> renders the component 1`] = `
<div
  className="col-sm-4 offset-4 text-center reset-cont"
>
  <br />
  <h3
    className="col-sm-8 offset-2"
  >
    Reset Password For Authors Haven
  </h3>
  <br />
  <div>
    <form>
      <div>
        <input
          className="col-sm-10 form-control offset-1"
          id="newpassword"
          placeholder="Enter new password"
          type="password"
        />
        <br />
        <input
          className="col-sm-10 form-control offset-1"
          id="confpassword"
          placeholder="Confirm password"
          type="password"
        />
      </div>
      <br />
      <div>
        <button
          className="btn col-sm-4 btn-dark"
          type="submit"
        >
          Submit
        </button>
      </div>
    </form>
    <br />
  </div>
</div>
`;

exports[`<PasswordReset /> renders the component 1`] = `
<div
  className="col-sm-4 offset-4 text-center reset-cont"
>
  <br />
  <h3
    className="col-sm-8 offset-2"
  >
    Reset Password For Authors Haven
  </h3>
  <br />
  <p>
    An email with a link to reset your password will be sent to your
  </p>
  <br />
  <div>
    <form>
      <div>
        <input
          className="col-sm-10 form-control offset-1"
          id="email"
          placeholder="Enter your email"
          type="email"
        />
         
      </div>
      <span
        className="text-danger"
      />
      <br />
      <div>
        <button
          className="btn col-sm-4 btn-dark"
          type="submit"
        >
          RESET
        </button>
      </div>
    </form>
    <br />
  </div>
</div>
`;
